manifests/overlays/k8s-env-e2e/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../../base/crd/overlays/local-dev
+- ../../base/gitops-namespace
+- ../../../appstudio-controller/config/default
+- ../../../backend/config/default
+- ../../../cluster-agent/config/default
+- ../../base/postgresql-staging
+# - ../../base/gitops-service-argocd
+
+patchesStrategicMerge:
+- backend-deployment-patch.yaml
+- cluster-agent-deployment-patch.yaml
```

Comment:
In order to reduce some of the duplication (which is what kustomize is good at!) between this overlay and the `k8s-env` overlay, I wonder if it would be possible to further reduce, by doing something like:

```yaml
resources:
- ../k8s-env # pull the same resources as the parent....

patchesStrategicMerge: # but patch the env as before

patchesStrategicMerge:
- backend-deployment-patch.yaml
- cluster-agent-deployment-patch.yaml
```

I _think_ this will work? WDYT?

Reply:
Done